src/engine.rs

```diff
@@ -414,3 +381,9 @@ pub fn run_treefmt_stdin(
 
     ret
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn output() {}
+}
```

Comment:
Did you want to add some test here?

Reply:
> Did you want to add some test here?

Oops, didn't mean to check that in, was just playing around